### ISSUE TITLE
The content and title of Expand Macro can not be changed via CKEditor after it was inserted

### ISSUE THREAD

**Issue by petrenkonikita112263** [xwikisas/xwiki-pro-macros#183]
**Prerequisites**:

- XWiki instance 13.10
- Latest version of Pro Macros
- Trial license

**Step to reproduce**:

1. Go to any Sandbox test page
2. Trigger the editing page via CKEditor and insert the Expand macro
3. Double-click on new macro and provide dummy content
4. Save page
5. Re-trigger editing page via CKEditor
6. Double-click on Expand macro

![msedge_bdR4SHazFs](https://github.com/xwikisas/xwiki-pro-macros/assets/56109799/e6795f15-cda8-4d42-bc3f-cf525cd50968)

**Actual result**:
The user can't change content or title of macro.
![image](https://github.com/xwikisas/xwiki-pro-macros/assets/56109799/5fbfe2e7-eb2b-4f27-9031-81fdc343a714)

**Expected result**:
Input fields of Expand macro are accessible and editable after double clicking (just like on first insert)
![image](https://github.com/xwikisas/xwiki-pro-macros/assets/56109799/5f608734-c122-46c4-8d2e-145d7b0995a8)

**Comment by mflorea** [comment]
@raphj , @petrenkonikita112263 yes this behavior is expected for macros that can be edited in-place. If the macro content or one of its parameters can be edited in-place then we don't show it anymore on the macro edit dialog.
